### DISCUSSION TITLE
Admin interface, strange translation

### DISCUSSION THREAD

**Discussion by PiemP** [OrchardCMS/OrchardCore#7279] [Q&A] [answered]
Hi all, 

I have downloaded the last translation from Crowdin site and the last code from the dev branch and I have a strange issue with the backend interfaces. This interfaces seems partially translated. I give you an image that show this:
![admin_localization](https://user-images.githubusercontent.com/2736009/96103151-9d9cf980-0ed7-11eb-99ed-be6a0b74cc56.jpg)

A week ago, the translation seem ok, I don't understand why this interfaces become partially translated after last code merge.
Could someone point me to right direction to fix this? 
Thank you.

**Comment by hishamco** [comment]
@PiemP please check the content translation po file, in `OrchardCore.Cms.Web\AppData\Localization\{Culture}` and see if the buttons translations are missing, otherwise will be a bug

**Reply by PiemP** [reply]
Thank you @hishamco for your answer. 

I have made the check and seem a bug. 
![po file](https://user-images.githubusercontent.com/2736009/96134658-9d165a00-0efb-11eb-8c3d-29747b4ba56b.jpg)

can I proceed to create an issue?
Thank you.

**Comment by hishamco** [comment, marked as answer]
@PiemP after a first look, seem the translations which available in OrchardCore.Translations not run the localization extractor tool to get the latest changes on the code, this seems to happened before `1.0.0` to get a fresh copy. @agriffard correct me if I'm wrong

**Reply by hishamco** [reply]
![Italian](https://user-images.githubusercontent.com/3237266/96528179-aea88a80-128a-11eb-8061-cea96263c5d6.png)

As I said the code change every day, every hour, every minute ;)

Try to add this in `OrchardCore.Contents.po`:

```
#. @T["Actions"]
#: OrchardCore.Contents\Views\Content.SummaryAdmin.cshtml:59
msgctxt "OrchardCore.Contents.Views.Content.SummaryAdmin"
msgid "Actions"
msgstr "Azioni"
```

**Reply by PiemP** [reply]
Thank you @hishamco.

**Reply by PiemP** [reply]
Sorry @hisamico, I continue to have some problem with the **Actions** button. It doesn't translated. I have checked in the `OrchardCore.Contents.po` file and I see this line of code: 

```
#. @T["Actions"]
#: OrchardCore.Contents\Views\Content.SummaryAdmin.cshtml:59
msgctxt "OrchardCore.Contents.Views.Content.SummaryAdmin"
msgid "Actions"
msgstr "Azioni"
``` 

I have checked `Content.SummaryAdmin.cshtml` and the button is in the line 59. Why this button is not translated? If you give me some tips I try to check by myself.
Thank you.

**Reply by hishamco** [reply]
No problem @PiemP feel free to ping me anytime, regarding your issue I remembered that I attached a screenshot to show you the valid translation, I'm not sure from that time if something changed in the view, I will check and come back to you ...

**Reply by PiemP** [reply]
I can confirm the previous solution... the translation working well. Sorry 😅
